test/oli_web/controllers/api/payment_controller_test.exs

```diff
@@ -5,9 +5,10 @@ defmodule OliWeb.PaymentControllerTest do
 
   alias Oli.Seeder
   alias OliWeb.Router.Helpers, as: Routes
+  alias Oli.Delivery.Paywall.Payment
 
   describe "payment controller tests" do
-    setup [:setup_session]
+    setup [:setup_session, :admin_conn]
```

Comment:
We should not modify the setup for all the tests inside the describe just to satisfy the one we are adding. What do you think about trying to replicate what the other tests are doing?